xls/examples/passthrough.x

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A simple proc that forwards the received information from
+// an input channel to an output channel.
+
+import std;
+
+proc Passthrough {
+    data_r: chan<u32> in;
+    data_s: chan<u32> out;
+
+    config(data_r: chan<u32> in, data_s: chan<u32> out) { (data_r, data_s) }
+
+    init { () }
+
+    next(tok: token, state: ()) {
+        let (tok, data) = recv(tok, data_r);
+        let tok = send(tok, data_s, data);
+    }
+}
+
+#[test_proc]
+proc PassthroughTest {
+    terminator: chan<bool> out;
+    data_s: chan<u32> out;
+    data_r: chan<u32> in;
+
+    config(terminator: chan<bool> out) {
+        let (data_s, data_r) = chan<u32>("data");
+        spawn Passthrough(data_r, data_s);
+        (terminator, data_s, data_r)
+    }
+
+    init { u32:0 }
+
+    next(tok: token, count: u32) {
+        const NUMBER_OF_TESTED_TRANSACTIONS = u32:10;
```

Comment:
what about using `NUMBER_OF_TESTED_TRANSACTIONS` as the state w/ an initial value of `10`?

Reply:
fixed